docs/sources/mimir/release-notes/v2.11.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Grafana Mimir version 2.11 release notes"
+menuTitle: "V2.11 release notes"
+description: "Release notes for Grafana Mimir version 2.11"
+weight: 1000
+---
+
+# Grafana Mimir version 2.11 release notes
+
+Grafana Labs is excited to announce version 2.11 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Sampled logging of errors in the ingester.** A high-traffic Mimir cluster can occasionally become bogged down logging high volumes of repeated errors. You can now reduce the amount of errors outputted to logs by setting a sample rate via the `-ingester.error-sample-rate` CLI flag.
+- **Add total request size instance limit for ingesters.** This limit protects the ingesters against requests that together may cause an OOM. Enable this feature by setting the `-ingester.instance-limits.max-inflight-push-requests-bytes` CLI flag.
```

Comment:
To make this work efficiently (without reading request to memory first), this needs to be used with `-ingester.limit-inflight-requests-using-grpc-method-limiter`.

Reply:
Got it, thanks!